src/web/core/src/main/resources/GeoServerApplication_ru.properties

```diff
@@ -12,18 +12,22 @@ AboutGeoServerPage.moreInformation = \u0411\u043e\u043b\u044c\u0448\u0435 \u0438
 
 # see comment in AboutGeoServerPage userMap, add the following lines at the end of
 # AboutGeoServerPage.information property
-# Also please take a moment to add # yourself to the User Map to show your support for GeoServer.
+# Also please take a moment to add \
+# yourself to the User Map to show your support for GeoServer.
 
 AboutGeoServerPage.introduction  = \u041f\u0440\u043e\u0435\u043a\u0442 GeoServer \u043f\u0440\u0435\u0434\u0441\u0442\u0430\u0432\u043b\u044f\u0435\u0442 \u0441\u043e\u0431\u043e\u0439 Java (J2EE) \u043f\u043e\u043b\u043d\u0443\u044e \u0442\u0440\u0430\u043d\u0437\u0430\u043a\u0446\u0438\u043e\u043d\u043d\u0443\u044e \u0440\u0435\u0430\u043b\u0438\u0437\u0430\u0446\u0438\u044e \u0441\u043f\u0435\u0446\u0438\u0444\u0438\u043a\u0430\u0446\u0438\u0438 Web Feature Server \u0438 \u0441\u043f\u0435\u0446\u0438\u0444\u0438\u043a\u0430\u0446\u0438\u0438 Web Coverage Server OpenGIS \u043a\u043e\u043d\u0441\u043e\u0440\u0446\u0438\u0443\u043c\u0430, \u0441\u043e \u0432\u0441\u0442\u0440\u043e\u0435\u043d\u043d\u044b\u043c Web Map Server.
+
 AboutGeoServerPage.gitRevision = \u0412\u0435\u0440\u0441\u0438\u044f Git
 AboutGeoServerPage.title         = \u041e GeoServer
+AboutGeoServerPage.usefulLinks   = Useful Links:
 AboutGeoServerPage.userMap       = \u041a\u0430\u0440\u0442\u0430 \u043f\u043e\u043b\u044c\u0437\u043e\u0432\u0430\u0442\u0435\u043b\u0435\u0439
 AboutGeoServerPage.wiki          = \u0412\u0438\u043a\u0438
 
 AbstractCoverageStorePage.dataSrcName = \u041d\u0430\u0438\u043c\u0435\u043d\u043e\u0432\u0430\u043d\u0438\u0435 \u0438\u0441\u0442\u043e\u0447\u043d\u0438\u043a\u0430 \u0434\u0430\u043d\u043d\u044b\u0445
 AbstractCoverageStorePage.description = \u041e\u043f\u0438\u0441\u0430\u043d\u0438\u0435
 
-AbstractDataAccessPage.cantGetDataStoreFactory = Can't find the factory for the given data store. Make sure the \nappropriate plugin is installed
+AbstractDataAccessPage.cantGetDataStoreFactory = Can't find the factory for the given data store. Make sure the \
```

Comment:
something went wrong here, "\n" replaced by "\" may result in a corrupt properties files